changelog.d/20231129_130322_franzavalla85_PLT_8828_remove_filter_getContracts_response.md

```diff
@@ -0,0 +1,7 @@
+### General
+
+- Allows to filter tags one by one on `getContracts` endpoint
+
```

Comment:
```suggestion

```

Reply:
The general category is for when a change doesn't fit in a particular category. For this, the runtime-rest-client category suffices